src/executor.ts

```diff
@@ -118,7 +124,11 @@ export class Executor {
   /**
    * Execute the root script.
    */
-  async execute(): Promise<Result<void, Failure[]>> {
+  async execute(): Promise<Result<ServiceMap, Failure[]>> {
+    // TOOD(aomarks) If we have any running services from a previous watch
```

Comment:
Do you plan on keeping services running across edits tot he wireit configs?

Reply:
Yep, as long as the fingerprint stays the same for a service, we can keep it running. So if you change something that would require the service to restart, it will do so. Otherwise no need to restart it.